### ISSUE TITLE
same result on F1, Accuracy, precision

### ISSUE THREAD

**Issue by wentinghome** [guillaumegenthial/tf_ner#68]
Hi,

I have applied a customized dataset in the few shot setting, the training contains 500, valid contains 1k, and the test contains 10k  examples,
Then, the result on F1, Accuracy, precision I got is exactly the same in the following.  Is anyone has a similar problem and any suggestions on this? Thanks so much!

-----------
Saving dict for global step 1879: acc = 0.9462044, f1 = 0.9462044, global_step = 1879, loss = 14.065788, precision = 0.9462044, recall = 0.9462044
